src/dbnode/tracepoint/tracepoint.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package tracepoint
+
+// The tracepoint package is used to store operation names for tracing throughout dbnode.
+// The naming convention is as follows:
+
+// `packageName.objectName.method`
+
+// If there isn't an object, use `packageName.method`.
+
+const (
+	// FetchTaggedSpanID is the operation name for the tchannelthrift FetchTagged path
+	FetchTaggedSpanID = "tchannelthrift/node.service.FetchTagged"
+
+	// QuerySpanID is the operation name for the tchannelthrift Query path
+	QuerySpanID = "tchannelthrift/node.service.Query"
+
+	// DBQueryIDs is the operation name for the db QueryIDs path
+	DBQueryIDs = "storage.db.QueryIDs"
```

Comment:
nit: first two vars in this file end with `...SpanID`, the last two don't. Mind sticking to one convention.

Reply:
Probably remove the `...SpanID` suffix, since someone will forget it at some point hah.